### PR TITLE
Testsuite fixes

### PR DESCRIPTION
**Pull request by ncounter**
## What does this PR change?

Referring to [this testsuite run](https://ci.suse.de/job/manager-Head-cucumber/ws/results/output-2019-05-10-00-39-45-2273.html)

Fixes:

Due to moving down in the order of features test scenarios, those tests were assuming `no system registered`, and that assumption is not true anymore.
  - Scenario: Completeness of the side navigation bar and the content frame
  - Scenario: Sidebar link destination for Systems => Virtual Systems
  - Scenario: Sidebar link destination for Systems => Out of Date
  - Scenario: Sidebar link destination for Systems => Non Compliant
  - Scenario: Sidebar link destination for Systems => Ungrouped
  - Scenario: Sidebar link destination for Systems => Recently Registered
  - Scenario: Sidebar link destination for Systems => Proxy
  - Scenario: Sidebar link destination for Systems => System Currency

Due to a message change to the bubble text on saving a formula, those tests were relying on the old text.
  - Scenario: Parametrize the formula on the minion
  - Scenario: Reset the formula on the minion
  - Scenario: Fill in and verify non-default values in group formula
  - Scenario: Clear values in group formula and verify the defaults again
  - Scenario: Fill in and verify mix of default and non-default values in group formula
  - Scenario: Verify that minion form inherits the values from group form
  - Scenario: Fill in and verify non-default values in minion formula
  - Scenario: Clear values in minion formula and verify that the pillar is set to group values
  - Scenario: Cleanup: remove "Testform" formula from "test-formula-group"
  - Scenario: Configure PXE part of DNS on the branch server
  - Scenario: Configure PXE part of DHCP on the branch server
  - Scenario: Parametrize TFTP on the branch server
  - Scenario: Parametrize vsFTPd on the branch server
  - Scenario: Configure PXE itself on the branch server
  - Scenario: Parametrize the Saltboot formula
  - Scenario: Cleanup: undo CNAME aliases

Due to a recent pattern change to use the menu in test scenarios, those steps were failing founding more than one link.
  - Scenario: Check the new bootstrapped minion in System Overview page
  - Scenario: Check new minion bootstrapped via XML-RPC in System Overview page
  - Scenario: Verify that minion bootstrapped with Salt key and packages
  - Scenario: Change the password to a new password
    - Scenario: Change the password to a new password
    - Scenario: Try an invalid password
  - Scenario: List systems by patch status via XML-RPC before patch
  - Scenario: Completeness of the onboarding page
  - Scenario: Minion is visible in the Pending section
  - Scenario: Reject and delete the pending key
  - Scenario: Accepted minion shows up as a registered system
  - Scenario: Change the schedule of task mgr-sync-refresh-default to weekly
  - Scenario: Change the schedule of task mgr-sync-refresh-default to monthly
  - Scenario: Change the schedule of task mgr-sync-refresh-default back to daily

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: it's about the testsuite

- [x] **DONE**

## Test coverage
- No tests: it's about the testsuite

- [x] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"

### CODE REVIEW

File: testsuite/features/min_salt_formulas.feature [hunk 1]
```diff
@@ -35,7 +35,7 @@ Feature: Use salt formulas
      And I select "French" in language field
      And I select "French (Canada)" in keyboard layout field
      And I click on "Save Formula"
-     Then I should see a "Formula saved!" text
+     Then I should see a text like "Formula saved*"
```
Comment (srbarrios) [inline testsuite/features/min_salt_formulas.feature]: Wildcard is no needed. The method already support it.
Please avoid non-natural language in Gherkin steps

Comment (srbarrios) [reply testsuite/features/min_salt_formulas.feature]: In fact, we don't need to use this other method, I'm not sure when that one was need.
If now we don't have "!", we only need to remove it from the step. So : Then I should see a "Formula saved" text

Comment (ncounter) [reply testsuite/features/min_salt_formulas.feature]: Addressed.

I would stay with the `text like` method anyway.
Before the change, the message was only `Formula saved!`, now it is `Formula saved. Apply the <a href=>.......`, and in the future the message maybe will slightly change again. The relevant thing is that the message contains the `Formula saved` text without depending on the place in the sentence or anything else.

Comment (Bischoff) [reply testsuite/features/min_salt_formulas.feature]: No, "like" is for regular expressions. You are not using one. It is substring match already.

Comment (ncounter) [reply testsuite/features/min_salt_formulas.feature]: In the end it turned out the `I should see a "..." text` implementation already works as a "partial match", so it is not needed at all the `text like` method. It is required when in the step we really have a `regex` in the text to match.
I am removing the `text like` usage then.

Comment (Bischoff) [reply testsuite/features/min_salt_formulas.feature]: Thank you very much.

Comment (srbarrios) [reply testsuite/features/min_salt_formulas.feature]: > In the end it turned out the `I should see a "..." text` implementation already works as a "partial match", so it is not needed at all the `text like` method. It is required when in the step we really have a `regex` in the text to match.
> I am removing the `text like` usage then.

Yep. That's what I tried to explain :) 
Better now. I'm not super strict on it anyway

**Pull request merged**